### ISSUE TITLE
ems_ss

### ISSUE THREAD

**Issue by afaqueali97** [afaqueali97/ems_issue_ss#1]

**Comment by afaqueali97** [comment]
Service Currently unavailable our technical team is working on it again later.

**Comment by afaqueali97** [comment]
Service Currently unavailable our technical team is working on it again later.
![image1](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/56e6ec48-3f2d-4db6-a73a-fefeaffad03d)

**Comment by afaqueali97** [comment]
Issue: App not installed as app isn't compatible with your phone.
Issue: Android version compatibility issue.
Issue: App not installed.

![image](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/0521b93c-b7ac-4a05-a15e-5e504d452ca6)

**Comment by afaqueali97** [comment]
Issue: IMEI number already assigned to other polling stations.

![image17](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/e5af39f8-97e0-45cc-9d16-c1f68debcb6d)

**Comment by afaqueali97** [comment]
Your Registration has been successful. you are requested to login to the app during Material distribution in the RO office
![image16](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/d05bcf64-a1bc-43fd-ae6f-aef0c5f65a84)

**Comment by afaqueali97** [comment]
IMEI number is already assigned, kindly contact the help Desk.

![image15](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/ab004e3b-e2fd-49ab-8a6c-561108e796a0)

**Comment by afaqueali97** [comment]
Blocked by your IT admin.
![image5](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/7e731304-46fe-496a-9f50-2e0adffe2aac)

**Comment by afaqueali97** [comment]
No internet Access. Please check your internet connection.
![image3](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/cd35bad1-739c-434b-a34d-1159274e0cee)

**Comment by afaqueali97** [comment]
Another issue has been reported regarding mock location.
![image4](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/3fc3afc7-9211-45a0-914e-fb0a0238645f)

**Comment by afaqueali97** [comment]
This application cannot run on rooted device.
![image14](https://github.com/afaqueali97/ems_issue_ss/assets/98380148/d2008418-565e-43ad-9d55-a70d5202ced1)
